src/tabs/character.js

```diff
@@ -29,11 +30,11 @@ export const Character = ({open, game, wearActions, panelsRef, setOpen, toggleOp
             <div className={classnames(styles['panel-section'], styles['name-placeholder'])} />
             <div className={classnames(styles['panel-section'], styles['main-stats'])}>
               <div className={styles['panel-row']}>
-                <h2>HP</h2>
+                <h2 id = "healthDisplay">HP </h2>
                 <progress value={61} />
               </div>
               <div className={styles['panel-row']}>
-                <h2>MP</h2>
+                <h2 id = "manaDisplay">MP</h2>
```

Comment:
Same spacing here, mentioned in other comment: https://github.com/webaverse/app/pull/2164/files#r779201048